test/success.test.js

```diff
@@ -48,14 +48,14 @@ test.serial('Post comments to related issues and MRs', async t => {
     .reply(200, [])
     .post(`/projects/100/merge_requests/1/notes`, {
       body:
-        ':tada: This MR is included in version 1.0.0 :tada:\n\nThe release is available on [GitLab release](https://gitlab.com/test_user%2Ftest_repo/-/releases/v1.0.0)\nYour **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:',
+        ':tada: This MR is included in version 1.0.0 :tada:\n\nThe release is available on [GitLab release](https://gitlab.com/test_user%2Ftest_repo/-/releases/v1.0.0)\nYour **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package: :rocket:',
```

Comment:
The URL here still looks wrong: `https://gitlab.com/test_user%2Ftest_repo/-/releases/v1.0.0`

Reply:
I think because we don't call the publish method with the method `decodeUriComponent`. We mock the created releases in this test. Will adapt the test data to avoid confusion.

Reply:
Ah, that's indeed very confusing. Would be great if you'd adapt the test data, thanks!

Reply:
Updated unit tests to use decoded URLs